plain_http_producer.go

```diff
@@ -28,7 +28,7 @@ func (c *plainHTTPMessageProducer) SendMessage(uuid string, message queueProduce
 	req, err := http.NewRequest("POST", c.config.Addr+"/notify", strings.NewReader(message.Body))
 	if err != nil {
 		logger.error(fmt.Sprintf("Error creating new request: %v", err.Error()))
```

Comment:
Can you please not log errors in the plain_http_producer? Follow the same pattern and create errors and return them. They will be logged in the message_forwarder.